sickbeard/versionChecker.py

```diff
@@ -185,12 +185,29 @@ def showupdate_safe(self):
             else:
                 logger.log(u"We can't proceed with the update. Shows are being updated", logger.DEBUG)
                 return False
+                
```

Comment:
Just counting queue items doesn't give a reliable result. You need to close the inflow of new queue items, or at least stop the processing of new queue items. Else, a daily search could start a.microsecond after you decided it is safe to upgrade